plugins/scaffolder-react/src/next/components/Stepper/Stepper.tsx

```diff
@@ -155,16 +158,15 @@ export const Stepper = (stepperProps: StepperProps) => {
   };
 
   const handleChange = useCallback(
-    (e: IChangeEvent) =>
-      setFormState(current => ({ ...current, ...e.formData })),
-    [setFormState],
+    (e: IChangeEvent) => {
+      setFormState(current => ({
+        ...current,
+        [`step${activeStep}`]: e.formData,
```

Comment:
I wonder if we should move this concat out into a function to make this easier to understand what's going on?

`[makeStepKey(activeStep)]` or something, and maybe it should be `step-${activeStep}` instead?

Reply:
I think that's a nice improvement. Using a small util method also make sure the keys are generated consistent